java/src/main/resources/arguments.yaml

```diff
@@ -28,10 +28,15 @@ ACCESS_TOKEN: null
 # =========================
 # Optional configurations:
```

Comment:
Would it be possible to add a separate TOPIC for change events, e.g. CHANGE_TOPIC? That way, we don't have to hardcode the Topic in ProcessChangeEventHeader.java.